ui/packages/consul-ui/app/utils/get-environment.js

```diff
@@ -23,7 +23,7 @@ export default function(config = {}, win = window, doc = document) {
             .startsWith('CONSUL_')
         );
     };
-    win.Scenario = function(str = '') {
+    win['Scenario'] = function(str = '') {
```

Comment:
Why this change?

Reply:
`tsc` doesn't complain about it if I do this, and it's the best solution I could find given I want to write as little typescript as possible. It's not great I know but also importantly: it's wrapped in a runInDebug, so this is all compiled out of our production build, so I'm ok with it.

It's funny as I just realized that this is actually cheating a little as I wanted to see what was possible without changing any existing code, so I'm happy either way if this goes in or not.